packages/core/src/odata/v2/batch-request-serializer.ts

```diff
@@ -29,9 +29,9 @@ import {
  * @param request - The request builder of the retrieve request.
  * @returns The request body.
  */
-export function toBatchRetrieveBodyV2(
+export async function toBatchRetrieveBodyV2(
   request: GetAllRequestBuilderV2<EntityV2> | GetByKeyRequestBuilderV2<EntityV2>
-): string {
+): Promise<string> {
```

Comment:
the signature of `serializeRequest` has not been changed, and still returns `string`, why you changed the signature of `toBatchRetrieveBodyV2`?

Reply:
Good catch. This was a left over. Will remove it.